pkg/apis/componentconfig/cloudcore/v1alpha1/default.go

```diff
@@ -150,6 +153,12 @@ func NewDefaultCloudCoreConfig() *CloudCoreConfig {
 				TLSStreamPrivateKeyFile: constants.DefaultStreamKeyFile,
 				StreamPort:              10003,
 			},
+			Router: &Router{
+				Enable:      true,
+				Address:     "0.0.0.0",
```

Comment:
Is it safe enough to use 0.0.0.0?

Reply:
refer other config, and it will use https in future